eth-types/src/geth_types.rs

```diff
@@ -49,12 +49,22 @@ impl From<TxType> for u64 {
 impl TxType {
     /// If this type is L1Msg or not
     pub fn is_l1_msg(&self) -> bool {
-        matches!(*self, TxType::L1Msg)
+        matches!(*self, Self::L1Msg)
     }
 
-    /// If this type is Eip155 or not
+    /// If this type is EIP-155 or not
     pub fn is_eip155_tx(&self) -> bool {
-        matches!(*self, TxType::Eip155)
+        matches!(*self, Self::Eip155)
+    }
+
+    /// If this type is EIP-1559 or not
+    pub fn is_eip1559_tx(&self) -> bool {
+        matches!(*self, Self::Eip1559)
+    }
```

Comment:
If this PR is handling only EIP-2930 related changes, better not include this here, so we can account for it in the EIP-1559 PR.

Reply:
Delete this `is_eip1559_tx` function in commit [cecb6c1](https://github.com/scroll-tech/zkevm-circuits/pull/1045/commits/cecb6c1c1a85b77f2d755980de9df4a39511a080).